packages/vike-react-styled-components/Wrapper.tsx

```diff
@@ -0,0 +1,17 @@
+export { Wrapper }
+
+import React, { type ReactNode } from 'react'
+import { StyleSheetManager } from 'styled-components'
+import { usePageContext } from 'vike-react/usePageContext'
+
+function Wrapper({ children }: { children: ReactNode }) {
+  const pageContext = usePageContext()
+  const { styleSheetManager } = pageContext.config
+
+  if (typeof window !== 'undefined' || styleSheetManager === null) return <>{children}</>
```

Comment:
Let's use a more robust test for the Vike extension: 
```ts
export function isBrowser() {
  // Using `typeof window !== 'undefined'` alone is not enough because some users use https://www.npmjs.com/package/ssr-window
  return typeof window !== 'undefined' && typeof window.scrollY === 'number'
  // Alternatively, test whether environment is a *real* browser: https://github.com/brillout/picocolors/blob/d59a33a0fd52a8a33e4158884069192a89ce0113/picocolors.js#L87-L89
}
```

Reply:
Thanks, I'll try it